### DISCUSSION TITLE
Drop down with inputs

### DISCUSSION THREAD

**Discussion by Ivann-2612** [directus/directus#13165] [Feature Requests]
Hello,

I have one question and I need help with Directus. I want to improve my app and add some input fields in the child relation when I open the collection of the child fields with inputs choose what input I want to import into the parent's collection. I have 7 input fields but I don't need always that 7 inputs, some of the situations are when I need just 3 inputs, etc…
I made a template with all 7 inputs and I need to choose how many I want depending on the situation. I saw in Directus, that they have like check box, and I want when I click on the chosen check box allowed to import that box with the input field.
I want to find some options when I should be able to add how many inputs I want.
For example, when I open the relational child component I want to click on some inputs and just those to import and use in the parent relational collection.

**Comment by Ivann-2612** [comment]
Maybe some option with + where should be able to add new input and dropdown where can I choose what I want to be added above drop-down in the parent component. When the drop-down is opened I have 7 options and I click on only three and I import those three inputs

**Comment by Ivann-2612** [comment]
Drop-down -> for exapmle option:
 name with input field like a key value pair
 lastname with input field
 age with input field
etc…
And what I choose to be added in item

**Comment by rijkvanzanten** [comment]
Do you have a visual example of what this interaction would look like? I'm having some trouble imagining what this looks like based on the given description 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
